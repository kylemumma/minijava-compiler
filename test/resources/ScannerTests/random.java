huk4iy289 uidbjsnma dw
efierof
usopjo  SDfsf F40-==-`./
 EWWL[\\\F
 /* hello */
// hello