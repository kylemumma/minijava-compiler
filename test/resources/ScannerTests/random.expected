ID(huk4iy289) ID(uidbjsnma) ID(dw) ID(efierof) ID(usopjo) ID(SDfsf) ID(F40) MINUS BECOMES BECOMES MINUS <UNEXPECTED(`)> DOT <UNEXPECTED(/)> ID(EWWL) LBRACK <UNEXPECTED(\)> <UNEXPECTED(\)> <UNEXPECTED(\)> ID(F)